boards/arm/bt510/bt510_defconfig

```diff
@@ -0,0 +1,37 @@
+# SPDX-License-Identifier: Apache-2.0
+
+CONFIG_SOC_SERIES_NRF52X=y
+CONFIG_SOC_NRF52840_QIAA=y
+CONFIG_BOARD_BT510=y
+
+# Enable MPU
+CONFIG_ARM_MPU=y
+
+# Enable RTT
+CONFIG_USE_SEGGER_RTT=y
+
+# enable GPIO
+CONFIG_GPIO=y
+
+# enable uart driver
+CONFIG_SERIAL=y
+
+# enable accelerometer & temperature sensor
+CONFIG_I2C=y
+CONFIG_SENSOR=y
+CONFIG_LIS2DH=y
+CONFIG_LIS2DH_TRIGGER_GLOBAL_THREAD=y
+CONFIG_LIS2DH_OPER_MODE_LOW_POWER=y
+CONFIG_SI7055=y
```

Comment:
Are you sure you want to enable these here? They will be enabled by default for all applications, including hello_world and blinky.

Reply:
PNG replaced with JPG to reduce physical size, both images reduced down to 250px in RST file.

The I2C sensors need to be enabled by default. I've created project overlay files to disable them for single threaded applications due to the underlying I2C drivers spawining additional threads.